src/runtime/TemporalObject.cpp

```diff
@@ -179,7 +200,7 @@ std::string TemporalObject::tzComponent(ExecutionState& state, std::string& isoS
 
 TemporalObject::DateTime TemporalObject::parseValidIso8601String(ExecutionState& state, std::string isoString, const bool parseTimeZone = false)
 {
-    DateTime dateTime = { 0, 1, 1, 0, 0, 0, 0, 0, 0, "", "", "", "" };
+    DateTime dateTime = { 0, 1, 1, 0, 0, 0, 0, 0, 0, new ASCIIString(), new TimeZone };
```

Comment:
Global variable `String::emptyString` represents an empty string.
So replace allocating a new empty string with this global emptyString value.